src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -183,23 +183,23 @@ trait MarathonConf
   )
 
   lazy val mesosAuthenticationPrincipal = opt[String]("mesos_authentication_principal",
-    descr = "Mesos Authentication Principal",
+    descr = "Mesos Authentication Principal.",
     noshort = true
   )
 
   lazy val mesosAuthenticationSecretFile = opt[String]("mesos_authentication_secret_file",
-    descr = "Mesos Authentication Secret",
+    descr = "Mesos Authentication Secret.",
     noshort = true
   )
 
   lazy val envVarsPrefix = opt[String]("env_vars_prefix",
-    descr = "Prefix to use for environment variables",
+    descr = "Prefix to use for environment variables.",
```

Comment:
Not necessarily part of this patch but this should be more specific, e.g.:

"Prefix to use for environment variables injected automatically into all started tasks."